### PR TITLE
Bug 1910783: Set termination_grace_period_seconds to 10 minutes

### PR DESCRIPTION
**Pull request by nirarg**

### CODE REVIEW

File: pkg/managers/vm/machine_scope.go [hunk 1]
```diff
@@ -242,7 +242,9 @@ func (s *machineScope) buildVMITemplate(namespace string) (*kubevirtapiv1.Virtua
 
 	ignitionSecretName := buildIgnitionSecretName(virtualMachineName)
 
-	template.Spec = kubevirtapiv1.VirtualMachineInstanceSpec{}
+	template.Spec = kubevirtapiv1.VirtualMachineInstanceSpec{
+		TerminationGracePeriodSeconds: func(src int64) *int64 { return &src }(terminationGracePeriodSeconds),
```
Comment (ravidbro) [inline pkg/managers/vm/machine_scope.go]: Why do you need a function here?
I see that until now, the struct was created empty, and then all fields were updated on that. Why behave differently in that case?

File: pkg/managers/vm/machine_scope.go [hunk 2]
```diff
@@ -46,10 +46,10 @@ const (
 	APIVersion                        = "kubevirt.io/v1alpha3"
 	Kind                              = "VirtualMachine"
 	mainNetworkName                   = "main"
+	terminationGracePeriodSeconds     = 600
+	providerIDFormat                  = "kubevirt://%s/%s"
 )
 
-const providerIDFormat = "kubevirt://%s/%s"
```
Comment (ravidbro) [inline pkg/managers/vm/machine_scope.go:51]: I understand the change, but I think it should be in a different commit

File: pkg/managers/vm/machine_scope.go (hunk 1 above)
Comment (nirarg) [reply pkg/managers/vm/machine_scope.go]: That is way to pass a pointer in one liner
I wasn't sure if using it was clear, so because of your comment, I changed the value to be defined before

File: pkg/managers/vm/machine_scope.go (hunk 2 above)
Comment (nirarg) [reply pkg/managers/vm/machine_scope.go:51]: Removed this change from the PR, will refactor later
